tests/db_migrator_input/config_db/dns-nameserver-expected.json

```diff
@@ -0,0 +1,8 @@
+{
+    "DNS_NAMESERVER|1.1.1.1": {
+        "state": "enabled"
+    },
+    "DNS_NAMESERVER|8.8.8.8": {
```

Comment:
It might be a good idea to add a test entry for IPv6 nameserver.

Reply:
Updated.